t/140-ssl-c-api.t

```diff
@@ -12,7 +12,7 @@ if ($openssl_version =~ m/built with OpenSSL (0|1\.0\.(?:0|1[^\d]|2[a-d]).*)/) {
     plan(skip_all => "too old OpenSSL, need 1.0.2e, was $1");
 
 } else {
-    plan tests => repeat_each() * (blocks() * 5 + 1);
+    plan tests => repeat_each() * 38;
```

Comment:
style: we'd better keep this style: `block() * 5 + N`.

Reply:
Sure. Reverted back. It took me a while to figure out what does `5` and `N` mean :)